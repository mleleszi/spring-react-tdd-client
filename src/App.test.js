import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App component", () => {
  it("should render app without errors", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
