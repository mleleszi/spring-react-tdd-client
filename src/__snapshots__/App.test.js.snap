// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render app without errors 1`] = `
<DocumentFragment>
  <div>
    hi
  </div>
</DocumentFragment>
`;
